app/modules/enduser_setup.c

```diff
@@ -397,18 +397,19 @@ static err_t close_once_sent (void *arg, struct tcp_pcb *pcb, u16_t len)
 /* ------------------------------------------------------------------------- */
 
 /**
- * Search String
+ * Get length of param
  *
- * Search string for first occurence of any char in srch_str.
+ * Search string for first occurence of deliemiter '&' or ' '.
```

Comment:
I think there are few naming/linguistic issues. All of them tiny but in combination this isn't easy to comprehend:

- I don't understand the second sentence in this comment.
- `@return -1 if...` - no longer true
-  `get_len(const char *str)` is extremely generic, can the function and the param be named more precisely?
-- `*str` is expected to be a "value&key=..." string, no?

Reply:
ok. To avoid more checkins:

``` c
/**
 * Get length of param value
 * 
 * This is being called with a fragment of the parameters passed in the 
 * URL for GET requests or part of the body of a POST request.
 * The string will look like one of these
 * "SecretPassword HTTP/1.1"
 * "SecretPassword&wifi_ssid=..."
 * "SecretPassword"
 * The string is searched for the first occurence of deliemiter '&' or ' '.
 * If found return the length up to that position.
 * If not found return the length of the string.
 *
 */
static int enduser_setup_get_lenth_of_param_value(const char *str)
```